src/UnityPackage/Assets/Editor/ServerSettings/ChainsafeServerSettings.cs

```diff
@@ -125,4 +137,30 @@ void OnGUI()
         }
         GUILayout.Label("Reminder: Your ProjectID Must Be Valid To Save & Build With Our SDK. You Can Register For One On Our Website At Dashboard.Gaming.Chainsafe.io", EditorStyles.label);
     }
+
+    public void WriteNetworkFile()
+    {
+        // declares paths to write our javascript files to
+        string path1 = "Assets/WebGLTemplates/Web3GL-2020x/network.js";
+        string path2 = "Assets/WebGLTemplates/Web3GL-MetaMask/network.js";
+
+        // writes data to the webgl default network file
+        StreamWriter writer1 = new StreamWriter(path1, false);
+        writer1.WriteLine("//You can see a list of compatible EVM chains at https://chainlist.org/");
+        writer1.WriteLine("window.networks = [");
+        writer1.WriteLine("  {");
+        writer1.WriteLine("id: " + PlayerPrefs.GetString("ChainID") + ",");
```

Comment:
These need to be indented with 4 spaces.

Reply:
tab only gave me 2 for some reason, odd

Reply:
moment

Reply:
fixed

Reply:
Not sure if I missed the fix, but I can suggest them here, would need to be done to label, token & rpc url probably 
```suggestion
        writer1.WriteLine("    id: " + PlayerPrefs.GetString("ChainID") + ",");
```